packages/component-model-user/entities/user/helpers/cached-request.js

```diff
@@ -0,0 +1,47 @@
+const superagent = require('superagent')
+const logger = require('@pubsweet/logger')
+const config = require('config')
+const ApiCache = require('./api-cache')
+
+const apiRoot = config.get('server.api.url')
+const elifeCache = new ApiCache(600)
+
+const cachedRequest = async (endpoint, query = {}) => {
+  const uri = apiRoot + endpoint
+  const req = superagent.get(uri)
+
+  // only had the header if its defined in config
+  const secret = config.get('server.api.secret')
+  if (secret) {
+    req.header.Authorization = secret
+  }
+  const hash = ApiCache.makeHash(uri + query, req)
```

Comment:
I just realised this is internal to the class and should not be public... rewriting a part of the class